src/code/z_actor_dlftbls.c

```diff
@@ -98,19 +98,47 @@ void ActorOverlayTable_LogPrint(void) {
 void ActorOverlayTable_FaultPrint(void* arg0, void* arg1) {
     ActorOverlay* overlayEntry;
     u32 overlaySize;
+    uintptr_t ramStart;
+    uintptr_t ramEnd;
+    u32 offset;
```

Comment:
does
```suggestion
    ptrdiff_t offset;
```
match? (signed type)
if yes that'd be prefered
if not, oh well :shrug:

Reply:
should overlaySize ideally be `size_t` too?

Reply:
ptrdiff_t does not appear to work: https://decomp.me/scratch/UTc4m

Reply:
oh well

> should overlaySize ideally be `size_t` too?

sure, if it matches

Reply:
it does not, int vs long thing I guess